DS and Algorithm/BST.py

```diff
@@ -10,15 +10,14 @@ def inorder_traversal(root):
         # Traverse the left subtree
         result += inorder_traversal(root.left)
 
-        # Visit the root node
         result.append(root.val)
+        result.append(root.val)  # Add the value to the list twice to get the node
```

Comment:
**Issue**: The value of the root node is appended twice to the result list, which could lead to incorrect results or data duplication in the inorder traversal of the binary search tree. <br> **Fix**: Remove the duplicate line that appends the root value twice. <br> **Code Suggestion**: 
 ```
 @@ -14 +14 @@
 -        result.append(root.val)  # Add the value to the list twice to get the node
 ```